src/components/Tooltip/TooltipRenderedOnPageBody.js

```diff
@@ -144,10 +163,33 @@ class TooltipRenderedOnPageBody extends React.PureComponent {
                         {this.props.text}
                     </Text>
                 </Text>
-                <View style={pointerWrapperStyle}>
-                    <View style={pointerStyle} />
-                </View>
-            </Animated.View>,
+            );
+        }
+
+        const isCustomContent = _.isFunction(this.props.renderTooltipContent);
+
+        return ReactDOM.createPortal(
+            <>
+                {/* If rendering custom content always render an invisible version of
+                    it to detect layout size changes, if the content updates. */}
+                {isCustomContent && (
+                    <View
+                        style={styles.invisible}
+                        onLayout={this.measureContent}
+                    >
+                        {this.props.renderTooltipContent()}
+                    </View>
+                )}
```

Comment:
I'm not sure about this, rendering the content twice does not seem as a good idea, any other way to detect the change?